extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/YoutubeParsingHelper.java

```diff
@@ -278,7 +289,7 @@ public static boolean isYoutubeChannelMixId(@Nonnull final String playlistId) {
     @Nonnull
     public static String extractVideoIdFromMixId(@Nonnull final String playlistId)
             throws ParsingException {
-        if (playlistId.startsWith("RDMM")) { // My Mix
+        if (isYoutubeMyMixId(playlistId)) { // My Mix
             return playlistId.substring(4);
 
         } else if (isYoutubeMusicMixId(playlistId)) { // starts with "RDAMVM" or "RDCLAK"
```

Comment:
The comments here can be removed, the method name or method itself now contains the information